src/EnergyPlus/OutputProcessor.cc

```diff
@@ -877,8 +877,8 @@ namespace OutputProcessor {
     }
 
     TimeStepType ValidateTimeStepType(EnergyPlusData &state,
-                                      std::string const &TimeStepTypeKey, // Index type (Zone, HVAC) for variables
-                                      std::string_view CalledFrom         // Routine called from (for error messages)
+                                      OutputProcessor::SOVTimeStepType const &TimeStepTypeKey, // Index type (Zone, HVAC) for variables
+                                      std::string_view CalledFrom                              // Routine called from (for error messages)
```

Comment:
ValidateTimeStepType is essentially just a converter from `SOVTimeStepType` to `TimeStepType`.  This seems a bit silly, but the core TimeStepType does not capture all the different possible entries, including System, Plant, and HVAC.  Trying to collapse them into just Zone and System is what led to the other PR being closed unmerged.  That can be investigated later.